telemetry/definitions/commonDefinitions.json

```diff
@@ -142,6 +147,63 @@
             "name": "aws_reportPluginIssue",
             "description": "Report an issue with the plugin"
         },
+        {
+            "name": "cloudwatchlogs_openGroup",
+            "description": "Open the CloudWatch Logs group window",
+            "metadata": [{ "type": "result" }]
+        },
+        {
+            "name": "cloudwatchlogs_openStream",
+            "description": "Open a CloudWatch Logs stream in the window",
+            "metadata": [{"type": "result"}]
+        },
+        {
+            "name": "cloudwatchlogs_downloadToFile",
```

Comment:
`downloadStreamToFile` for consistency with other *stream operations?

Reply:
Sure, changed